core/src/main/java/uk/ac/bbsrc/tgac/miso/core/util/TaxonomyUtils.java

```diff
@@ -58,44 +56,32 @@
  * @since 0.1.4
  */
 public class TaxonomyUtils {
-  private static final Logger log = LoggerFactory.getLogger(TaxonomyUtils.class);
 
-  private static final String ncbiEntrezUtilsURL = "http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?";
+  private static final String ncbiEntrezUtilsURL = "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=taxonomy&term=";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)